src/jobrunner.rs

```diff
@@ -235,7 +235,7 @@ impl JobRunner {
             // Should we check the queue? This could be because we were previously unable to empty
             // it fully, or because the HTTP server has told us that it's put more jobs there.
             // However, it's only worth us checking the queue (which requires a lock) if there's
-            // space for us to run further jobs.
+            // space for us to run further jobs. Note that this check *must* come after
```

Comment:
Is that a partial comment, or does it just need a full-stop?

Reply:
Oops, I got half way through writing this, and realised I could solve the problem another way. So this is now misleading. Removed in https://github.com/softdevteam/snare/pull/17/commits/caadd495cdf6ad95ca0a07bd3a844da0122d7f11.